cmake/std/atdm/apps/empire/EMPIRETrilinosEnables.cmake

```diff
@@ -1,3 +1,12 @@
+IF ("$ENV{ATDM_CONFIG_BUILD_TYPE}" STREQUAL "DEBUG")
+	ATDM_SET_ENABLE(PanzerAdaptersSTK_CurlLaplacianExample-ConvTest-Quad-Order-4_DISABLE ON)
+	ATDM_SET_ENABLE(PanzerAdaptersSTK_MixedPoissonExample-ConvTest-Hex-Order-3_DISABLE ON)
+ENDIF()
+
+IF (ATDM_USE_CUDA)
+	ATDM_SET_ENABLE(Sacado_ENABLE_HIERARCHICAL_DFAD ON)
```

Comment:
@bbraunj, just noted that this logic is already in:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/std/atdm/ATDMDevEnvSettings.cmake#L289

Can you remove that redundant setting as well?

Reply:
Ah okay. I missed that. Looks like there's really only a couple flags being added then. I have updated the branch.